### ISSUE TITLE
[A11y] Collection Landing - facets (link purpose, link function, widgets, ...)

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1834]
### Operating System

Windows

### Browser

Edge

### Affected page

Collection Show

### Admin user?

Yes

### Issue Description

The facet links are really behaving like disclosure buttons (but aren't coded properly) since they are used to show and hide facet filtering options.  The text, e.g., "Format", doesn't describe the purpose, the link is missing widget semantics necessary for a disclosure, it doesn't function as a disclosure, etc.

This is very similar to #1797.  There are a lot of failures on these links that don't make sense to write up as individual tickets.

### Suggested Fix

The easiest fix is probably to just implement these as disclosure buttons: https://www.w3.org/TR/wai-aria-practices/#disclosure

I believe part of the problem is the div being the clickable entity, but not being an actual control.  Looking at the code I'm seeing that the surrounding divs actually have ARIA attributes on them.  But those aren't seen by screen readers since the divs aren't real controls and never get focus.

**Comment by jechols** [comment]
QA: need expertise to verify disclosure pattern

**Comment by jechols** [comment]
I believe @CGillen was going to use an accordion here, not a disclosure, so testing should adjust based on the final implementation

**Comment by CGillen** [comment]
Fixed by #1899

**Comment by jechols** [comment]
Returning for a quick fix. Everything else looks good, but the pattern requires this:

>  The accordion header button element has aria-controls set to the ID of the element containing the accordion panel content.
